pkg/logging/logging.go

```diff
@@ -0,0 +1,13 @@
+package logging
+
+import (
+	"context"
+
+	"github.com/go-logr/logr"
```

Comment:
packethost/pkg/log actually came out of pbnj and then was tweaked. Most other services here are using pkg/log lets use it here too and then later we can move everything to use logr and have pkg/log implement it.